src/calculateRentalCost.js

```diff
@@ -5,6 +5,22 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  const price = 40;
+  const basePrice = days * price;
+
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISCOUNT = 20;
+
+  if (days >= LONG_TERM) {
+    return basePrice - LONG_TERM_DISCOUNT;
+  } else if (days >= SHORT_TERM) {
+    return basePrice - SHORT_TERM_DISCOUNT;
+  }
```

Comment:
According to the checklist, you should avoid using `else` after `if` with `return`. The `else if` block is unnecessary here, as the `return` statement inside the first `if` block will exit the function if the condition is met. You can remove the `else` keyword and simply use `if` for the second condition check.